src/Core/Localization/CLDR/LocaleData.php

```diff
@@ -170,6 +177,10 @@ public function overrideWith(LocaleData $localeData)
             }
         }
 
+        if (null !== $localeData->getTerritories()) {
+            $this->territories = array_merge($this->territories, $localeData->getTerritories());
```

Comment:
```suggestion
            $this->setTerritories(array_merge($this->territories, $localeData->getTerritories()));
```

Reply:
To be consistent with https://github.com/PrestaShop/PrestaShop/pull/21680

Reply:
Fixed.